mflowgen/icovl/pre-flowsetup/configure.yml

```diff
@@ -15,9 +15,11 @@ outputs:
   - icovl-cells.lef
   - dtcd-cells.lef
   - bumpcells.lef
+  - sealring.lef
```

Comment:
This entire script is unnecessary... the flowsetup setup.tcl accepts all lef files in the adk as inputs. ([see here](https://github.com/cornell-brg/mflowgen/blob/71ff42c17ddb54eb269d2b967101582869858098/steps/cadence-innovus-flowsetup/setup.tcl#L117))